.github/workflows/flutter.yml

```diff
@@ -82,6 +94,9 @@ jobs:
             web)
             flutter build web
             ;;
+            linux)
+            flutter build linux
```

Comment:
the other builds have a `--release`, do we need this here?

Reply:
Yep good catch